docs/reference/remap/functions/timestamp.cue

```diff
@@ -32,9 +32,9 @@ remap: functions: timestamp: {
 			title: "Declare a timestamp type"
 			input: log: timestamp: "2020-10-10T16:00:00Z"
 			source: #"""
-				timestamp(.timestamp)
+				ok, err = timestamp(.timestamp)
 				"""#
-			return: input.log.timestamp
+			return: "function call error for \"timestamp\" at (10:31): expected \"timestamp\", got \"string\""
```

Comment:
There's no useful way to test this function (I'm not even sure if there's any reason to keep it), since you can never get a timestamp type directly from an event, you'll either have to use literal timestamps (`t'...'`) or use one of the timestamp coercion functions (`to_timestamp` or `parse_timestamp`).

I'll leave this as is for now, but we can either update the test suite to ignore tests without return values (and thus remove the return value here), or do something else.